From f72dbb58cdd9af0b783bac5a5d41bed585c8bc08 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Mon, 17 Sep 2018 09:31:24 +0300
Subject: [PATCH 4666/5242] MLK-19573-1: ASoC: fsl: dsd: make
 fsl_get_pins_state inline

commit  28a915a1b750d7e4a457e93a2ab5219a4941e986 from
https://source.codeaurora.org/external/imx/linux-imx.git

Make fsl_get_pins_state function inline.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit badcb97ebd8c0aae89f76e979bcc801be35c7400)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_dsd.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_dsd.h b/sound/soc/fsl/fsl_dsd.h
index bc14963..a45f314 100644
--- a/sound/soc/fsl/fsl_dsd.h
+++ b/sound/soc/fsl/fsl_dsd.h
@@ -29,7 +29,7 @@ static bool fsl_is_dsd(struct snd_pcm_hw_params *params)
 	}
 }
 
-static struct pinctrl_state *fsl_get_pins_state(struct pinctrl *pinctrl,
+static inline struct pinctrl_state *fsl_get_pins_state(struct pinctrl *pinctrl,
 	struct snd_pcm_hw_params *params, u32 bclk)
 {
 	struct pinctrl_state *state = 0;
-- 
1.7.9.5

